pkg/kubectl/cmd/util/helpers.go

```diff
@@ -290,6 +293,7 @@ func GetWideFlag(cmd *cobra.Command) bool {
 func GetFlagBool(cmd *cobra.Command, flag string) bool {
 	b, err := cmd.Flags().GetBool(flag)
 	if err != nil {
+		debug.PrintStack()
```

Comment:
> debugging cruft.

While I'm here, this is why we should bind flags, because chasing the fatal is massive pain.